okhttp/src/main/java/okhttp3/internal/http2/Http2Writer.kt

```diff
@@ -100,7 +100,7 @@ class Http2Writer(
     hpackWriter.writeHeaders(requestHeaders)
 
     val byteCount = hpackBuffer.size
-    val length = Math.min((maxFrameSize - 4).toLong(), byteCount).toInt()
+    val length = minOf((maxFrameSize - 4).toLong(), byteCount).toInt()
```

Comment:
the toLong seems weird here given there is toInt, is there better way to express this?

Reply:
How about` maxFrameSize - 4L`?